Svc/PolyDb/docs/sdd.md

```diff
@@ -67,11 +67,11 @@ No dictionaries
 
 ## 4. Module Checklists
 
-Document | Link
--------- | ----
-Design Checklist | [Link](Checklist_Design.xlsx)
-Code Checklist | [Link](Checklist_Design.xlsx)
-Unit Test Checklist | [Link](Checklist_Unit_Test.xls)
+Checklist |
+-------- |
+[Design](Checklist_Design.xlsx) |
+[Code](Checklist_Code.xlsx) |
+[Unit Test](Checklist_Unit_Test.xls) |
```

Comment:
This is one of the tables where there was a bug in the links. The second row said `Code` but had a `Link` to `Design`.